### DISCUSSION TITLE
Finish treatment even on changing page

### DISCUSSION THREAD

**Discussion by Sylvain-Caron** [odoo/owl#1475] [Q&A] [answered]
Hello, I would like some information

I have several functions in my component including requests, and even if I change pages I would like the processes to finish in the background so as not to have any interruptions

Do you have any information for me on the possibilities to manage this ?

**Comment by sdegueldre** [comment, marked as answer]
I'm not sure how this involves owl at all.

**Reply by Sylvain-Caron** [reply]
yes i think too, i will check if i can get more information on the Odoo side
